### PR TITLE
[Arms Warrior] Add arms warrior covenants abilities to statistics

### PR DESCRIPTION
**Pull request by Chasson1992**
Adding the covenant abilities to the arms warrior ability file in order to display spell statistics. Turns out that condemn has multiple spell IDs for spec and talent choices (similar to Execute), so I updated the warrior covenant ability file as well.

For those less familiar with warrior, execute spell gets replaced by Condemn (Venthyr covenant ability) so you'll notice I added an enable check on execute to check for the combatants covenant. 

Added myself to the contributors tab, updated the CHANGELOG for the spec.

Tested changes using Warcraft logs of a player from each covenant and a few fury warriors.

### CODE REVIEW

File: src/parser/warrior/arms/CHANGELOG.js [hunk 1]
```diff
@@ -1,7 +1,8 @@
-import { Abelito75 } from 'CONTRIBUTORS';
+import { Abelito75, Carrottopp } from 'CONTRIBUTORS';
 import { change, date } from 'common/changelog';
 
 export default [
   change(date(2020, 12, 17), 'Added some shadowlandisms.', Abelito75),
   change(date(2020, 12, 17), 'Removed all BFAisms.', Abelito75),
+  change(date(2020, 12, 25), 'Add covenant abilites to statistics', Carrottopp),
```
Comment (abelito75) [inline src/parser/warrior/arms/CHANGELOG.js]: this should go on the top of the list

File: src/parser/warrior/arms/modules/Abilities.js [hunk 2]
```diff
@@ -72,6 +73,15 @@ class Abilities extends CoreAbilities {
         gcd: {
           base: 1500,
         },
+        enabled: !combatant.hasCovenant(COVENANTS.VENTHYR.id),
+      },
+      {
+        spell: [SPELLS.CONDEMN, SPELLS.CONDEMN_MASSACRE, SPELLS.CONDEMN_FURY, SPELLS.CONDEMN_FURY_MASSACRE],
```
Comment (abelito75) [inline src/parser/warrior/arms/modules/Abilities.js]: Arms doesn't need fury's condemn

**Review by abelito75 — CHANGES_REQUESTED** [review]

File: src/parser/warrior/arms/CHANGELOG.js (hunk 1 above)
Comment (Chasson1992) [reply src/parser/warrior/arms/CHANGELOG.js]: Updated in latest commit

File: src/parser/warrior/arms/modules/Abilities.js (hunk 2 above)
Comment (Chasson1992) [reply src/parser/warrior/arms/modules/Abilities.js]: Updated in latest commit

**Review by abelito75 — APPROVED** [review]

**Pull request merged**